src/constants/colors.ts

```diff
@@ -0,0 +1,36 @@
+export const ASSET = '#06B6D4';
+export const LIABILITY = '#FF6600';
+export const INCOME = '#22C55E';
+export const EXPENSE = '#EF4444';
```

Comment:
would be good to find a way to export the tailwindcss color variables into this so we can re-use